media/css/firefox/mobile/mobile.scss

```diff
@@ -163,6 +163,7 @@ main .mzp-l-content {
     @include image-replaced;
     background-position: top center;
     background-repeat: no-repeat;
+    background-size: contain;
```

Comment:
The fix works on low-res screens (nice!), but the `at2x()` mixin is creating another `background-size` property for high-res screens that takes precedence. You may want to try and pass `contain` to the mixin above instead of `auto, 100%`?

Reply:
There's no need for this duplicate property now that you've specified `contain` in the mixin. Please remove this line.

Reply:
You are right. I thought that this was needed for the two different cases (low-res, high-res) but as the image itself is included by the `at2x()` mixin it's not necessary.